### ISSUE TITLE
Quick Noob question: How do I add <"Photoshop"> tag correctly?

### ISSUE THREAD

**Issue by Frozenkex** [Bionus/imgbrd-grabber#713]
Basically , i want to rename my files to include photoshop tag if the image has it, but I think I'm not doing it right.

**Comment by elreymu** [comment]
I would do that by making a token and call it "photoshop" or anything you want.
then in the tag list of that token i would put "photoshop" there
then in the filename you can add the photoshop token like %photoshop%

**Comment by Frozenkex** [comment]
That worked well, thanks. According to wiki (Tag Conditionals) i thought it would simply work.

**Comment by Bionus** [comment]
What elreymu said is indeed another solution.
Another would be a conditional filename triggered by the presence of the "protoshop" tag.

Ass for using tag conditionals, using `<"photoshop" >%md5%.%ext%` for example should work, generating `photoshop %md5%.%ext%` if the tag is present, and only `%md5%.%ext%` otherwise.

From the title of the issue I think you already found that out, though. Did it not work? I tried it just it case and worked on my end.

**Comment by Frozenkex** [comment]
I think I realized why it didn't work for me initially. It seems that it is case-sensitive and i tried to use the tag with capital letter in my re-naming process. Normally this isn't an issue when you write down tokens in destination name, because when you press save it adjusts the case automatically to the correct one. Tested this by trying to rename files and changing first letters of tokens to capital letters, which made these tags completely ignored.
